examples/quickstart/jupyter-notebooks/druidapi/README.md

```diff
@@ -36,21 +39,25 @@ At present, the best way to use `druidapi` is to clone the Druid repo itself:
 git clone git@github.com:apache/druid.git
 ```
 
-`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`
+`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
+From this directory, install the package and its dependencies with pip using the following command:
 
-Eventually we would like to create a Python package that can be installed with `pip`. Contributions
-in that area are welcome.
+```
+pip install .
+```
 
-Dependencies are listed in `requirements.txt`.
+Verify your installation by checking that the following commands run in Python without error:
 
-`druidapi` works against any version of Druid. Operations that exploit newer features obviously work
-only against versions of Druid that support those features.
+```python
+import druidapi
+druid = druidapi.jupyter_client('http://localhost:8888')
+```
 
-## Getting Started
+## Getting started
 
 To use `druidapi`, first import the library, then connect to your cluster by providing the URL to your Router instance. The way that is done differs a bit between consumers.
 
-### From a Tutorial Jupyter Notebook
+### From a tutorial Jupyter Notebook
```

Comment:
Always odd that we don't use title case in titles... Is Notebook a proper noun here? Or, should this be:

> From a tutorial Jupyter notebook

Reply:
We use sentence case for headings as per the Google style guide: https://developers.google.com/style/capitalization. Will update `notebook` to lowercase n.